sdk/batch/Microsoft.Azure.Management.Batch/src/Generated/SdkInfo_BatchManagement.cs

```diff
@@ -1,46 +0,0 @@
-
```

Comment:
this file should still be generated, can you please include it?

Reply:
Why do we need it? It's internal and not referenced anywhere.

Reply:
Honestly, not sure - It may be used by some tooling to discover api-versions, I do  know that it is required, and should be generated as part of running generation.

Reply:
Do you know if there's a flag I need to set when calling AutoRest or something to get it to generate? Not sure why it's not.